skcosmo/feature_selection/simple_fps.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import numbers
+
+from sklearn.utils.validation import NotFittedError
+from ._greedy import GreedySelector
+
+
+class SimpleFPS(GreedySelector):
```

Comment:
Do I understand correctly that right now warm_start only works with the same dataset? Don't we need to extend the functionality to different datasets? We discuss it previously, but I didn't remember the final decision.